docker/release/Dockerfile

```diff
@@ -63,6 +64,7 @@ ENV OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_LOCAL_REPOSITORY="/usr/
 ENV OSTK_PHYSICS_ENVIRONMENT_EPHEMERIDES_SPICE_MANAGER_LOCAL_REPOSITORY="/usr/local/share/OpenSpaceToolkit/Physics/environment/ephemerides/spice"
```

Comment:
Does 
```
COPY --from=us.gcr.io/loft-orbital-public/software/open-space-toolkit/data:0.3.0 /open-space-toolkit/physics /usr/local/share/OpenSpaceToolkit/Physics
```
this line do anything?

Reply:
ok thanks for the explanation, so it can be removed then?

Reply:
Yeah it can be removed, but it should be replaced with a `git clone` similar to the other repos

Reply:
Didn't realize it was in this file still. Good catch!